src/js/components/visualization/Visualization.jsx

```diff
@@ -93,6 +93,68 @@ class Visualization extends Component {
       typeGroup
     }
   }
+  
+  renderError (error) {
+    const {visualization} = this.props
+    const {name} = visualization
+    let text = error.message
+    
+    try {
+      text = JSON.parse(error.message)
```

Comment:
If we need to parse the error we get back, than that should probably be done in the action itself.